cargo/cargo_build_script.bzl

```
@@ -121,6 +117,13 @@ def _build_script_impl(ctx):
         if cc_toolchain.sysroot:
             env["SYSROOT"] = cc_toolchain.sysroot
 
+    # In form build scripts of rustc flags
+    # https://github.com/rust-lang/cargo/issues/9600
+    env["CARGO_ENCODED_RUSTFLAGS"] = " ".join([
```

As per the docs at https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-build-scripts
```suggestion
    env["CARGO_ENCODED_RUSTFLAGS"] = "\x1f".join([
```